resources/qml/NotificationWarning.qml

```diff
@@ -29,7 +29,7 @@ Item {
             anchors.rightMargin: 10
             anchors.bottom: parent.bottom
             color: Nheko.theme.red
-            text: qsTr("You will be pinging the whole room")
+            text: qsTr("You are about to ping the whole room")
```

Comment:
I don't think that is much of an improvement. "ping" is not really an understandable word? Maybe "You will be causing a notification for everyone in the room"?

Reply:
Or "you are about to notify the whole room"?

Reply:
Sounds weird, but my English isn't great :D

Reply:
Well, "will be causing" is not really how Americans talk ;)

Reply:
I guess, but I wanted to emphasize, that you are responsible for a lot of pings. I guess this is fine though, thank you!